atm.js

```diff
@@ -1,3 +1,89 @@
+
+
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
+
+//Pseudo Code for Deposit overall function
+
+// Make Balance an integer without the $
+
+// var zeroBalanceText = $("#checking .balance").text();
+// var zeroBalanceNoDSign =  zeroBalanceText.split("$")[1];
+// var zeroBalanceInt = parseInt(zeroBalanceNoDSign);
+
+
+
+
+//Code that Workds
+
+
+var depositChecking = $("#checking .deposit");
+depositChecking.on("click", function(){
+      var prevBalance = $("#checking .balance").val().replace("$","");
+      var newEntry = $("#checking .input").val();
+
+        var newBalance = prevBalance + newEntry;
+
+        $("#checking .balance").html("$" + newBalance)
+
+    })
+
+
+var withdrawChecking = $("#checking .withdraw")
+
+withdrawChecking.on("click", function(){
+  var prevBalance = $("#checking .balance").val().replace("$","");
```

Comment:
This code looks very similar to what you have in your deposit checking listener. Perhaps this is an opportunity for refactoring. Think about how you could use `this` to do that...